src/ui/pages/Connections/Connections.scss

```diff
@@ -25,6 +25,10 @@
 
   > ion-content {
     overflow: initial;
+
+    ion-list {
+      background: transparent;
+    }
```

Comment:
fixes connection list css issue for android

Reply:
Please @iFergal revert this one, I'll fix it in a separate ticket.